spec/syncSpec.js

```diff
@@ -105,53 +113,56 @@ describe('Sync', function() {
 
   it('should delete', function() {
     return manage(datasetId)
-    .then(doCreate(datasetId, testData))
-    .then(doDelete())
-    .then(doRead(datasetId))
-    .catch(function(err) {
-      expect(err).toEqual('unknown_uid');
-    });
+      .then(doCreate(datasetId, testData))
+      .then(function withResult(res) {
+        const uid = res.uid;
+        return doDelete(datasetId, uid)
+        .then(doRead(datasetId, uid).catch(function(err) {
+          expect(err).toEqual('unknown_uid');
+          console.log(err);
```

Comment:
Sorry about that, was in a rush and did not notice I left it there. Will remove it. Thanks